bigquery/stackoverflow.rb

```diff
@@ -0,0 +1,42 @@
+# Copyright 2017 Google LCC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A short sample demonstrating how to make a BigQuery API request
+
+# [START bigquery_simple_app_all]
+# [START bigquery_simple_app_deps]
+require "google/cloud/bigquery"
+# [END bigquery_simple_app_deps]
+
+# [START bigquery_simple_app_client]
+# This uses Application Default Credentials to authenticate.
+# @see https://cloud.google.com/bigquery/docs/authentication/getting-started
+bigquery = Google::Cloud::Bigquery.new
+# [END bigquery_simple_app_client]
+
+# [START bigquery_simple_app_query]
+sql = "SELECT " +
```

Comment:
One last nit, could you align the `=` for sql and results assignments.

Reply:
sure, done.